recipes/python-neo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "neo" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: python-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: python-{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 98e7e8948158f4492fc94d9d44367c91
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.3.0
+    - quantities >=0.9.0
+
+  run:
+    - python
+    - numpy >=1.3.0
+    - quantities >=0.9.0
+
+test:
+  imports:
+    - neo
+    - neo.core
+    - neo.io
+    - neo.test
+    - neo.test.iotest
+
+about:
+  home: http://neuralensemble.org/neo
+  license: BSD 3-Clause
+  summary: Neo is a package for representing electrophysiology data in Python, together with support for reading a wide range of neurophysiology file formats.
```

Comment:
Yeah, was also thinking this is too long, but it had come from their [description]( https://github.com/NeuralEnsemble/python-neo/blob/3f9a5a62fda9b3f4d19a51f44b037338cc55d21a/setup.py#L22 ). Will take some liberties with shortening it.